integration/spark/src/main/scala/org/apache/carbondata/spark/rdd/CarbonDataRDDFactory.scala

```diff
@@ -1111,25 +1131,32 @@ object CarbonDataRDDFactory extends Logging {
     val table = org.apache.carbondata.core.carbon.metadata.CarbonMetadata.getInstance
       .getCarbonTable(carbonLoadModel.getDatabaseName + "_" + carbonLoadModel.getTableName)
     val metaDataPath: String = table.getMetaDataFilepath
-    val carbonLock = CarbonLockFactory
+    val carbonCleanFilesLock = CarbonLockFactory
       .getCarbonLockObj(table.getAbsoluteTableIdentifier.getCarbonTableIdentifier,
-        LockUsage.METADATA_LOCK
+        LockUsage.CLEAN_FILES_LOCK
       )
     try {
-      if (carbonLock.lockWithRetries()) {
+      if (carbonCleanFilesLock.lockWithRetries()) {
+        logger.info("Clean files lock has been successfully acquired.")
         deleteLoadsAndUpdateMetadata(carbonLoadModel,
           table,
           partitioner,
           hdfsStoreLocation,
           isForceDeletion = true)
       }
+      else {
+        val errorMsg = "Clean files request is failed for " + carbonLoadModel.getDatabaseName +
```

Comment:
"Clean file request is running in the background, please try after some time". u[date the comment since already we know the operation